### PR TITLE
Feat/cypress paired page

### PR DESCRIPTION
**Pull request by colganmeanor**
What (if any) features are you implementing?
- implementing cypress testing for pairedpage component. 

What (if anything) did you refactor?
- refactored info on pairingForm component to fix broken tests for homepage
- refactored info in homepage.js test to fix broken tests for homepage
Were there any issues that arose?
- Cypress Syntax - what even is it? Who knows? I dont. 
Is there anything that you need from your teammate?
- Please review and let me know if any issues to resolve before merging. 
Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: cypress/fixtures/sampleDrink.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "strMeasure11": null,
+        "strMeasure12": null,
+        "strMeasure13": null,
+        "strMeasure14": null,
+        "strMeasure15": null,
+        "strImageSource": "https://www.instagram.com/p/CLj-ofvF0tR/",
+        "strImageAttribution": null,
+        "strCreativeCommonsConfirmed": "No",
+        "dateModified": null
+    }
```
Comment (raz-joan) [inline cypress/fixtures/sampleDrink.json:55]: Depending on time, we could remove this or try to use it in an intercept 👍🏻

File: cypress/integration/homepage.js [hunk 2]
```diff
@@ -32,47 +32,47 @@ describe('Board Game Bartender Home Page', () => {
...
 
-        cy.get('[data-cy=liquors] option').eq(6)
+        cy.get('[data-cy=liquor-choice] option').eq(6)
             .should('have.value', 'Scotch')
 
-        cy.get('[data-cy=liquors] option').eq(7)
+        cy.get('[data-cy=liquor-choice] option').eq(7)
             .should('have.value', 'Non Alcoholic')
     })
 })
```
Comment (raz-joan) [inline cypress/integration/homepage.js:78]: THANK you for fixing these tests @colganmeanor !!

File: src/Components/PairingForm.js [hunk 3]
```diff
@@ -37,19 +37,16 @@ const PairingForm = () => {
...
             <img src={require('../assets/games-drinks-friends.jpeg')} />
             </div>
-            
-            <form className='game-liquor-input' data-cy='pairing-form'>
-                <label htmlFor='game-choice' className='game-input'>
-                    
-                    <input className='game-dropdown' data-cy='game-dropdown' id='game-choice' placeholder='Choose Your Game!' list='games' onChange={(event) => dispatch(setGameName(event.target.value))}/>
+            <form className='game-liquor-input pairing-form' data-cy='pairing-form'>
+                <label htmlFor='game-choice' data-cy='pairing-form' className='game-input'>    
+                    <input className='game-dropdown' data-cy='game-choice' id='game-choice' placeholder='Choose Your Game!' list='games' onChange={(event) => dispatch(setGameName(event.target.value))}/>
```
Comment (michelecomfort) [inline src/Components/PairingForm.js:42]: This data-cy label might need to be changed in favoritePage cypress testing so the tests are all passing, I believe.

File: cypress/integration/pairedpage.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+        .go('back')
+
+        .get('h3')
+        .contains('Catan: Explorers & Pirates')
+
+        .get('h3')
+        .contains('Spice 75')
+    })
+
```
Comment (raz-joan) [inline cypress/integration/pairedpage.js:82]: @colganmeanor these are awesome tests! I really appreciate that you used different url's in the .visits(). Really clever!

**Review by michelecomfort — COMMENTED** [review]
Nice cypress testing!!

**Review by raz-joan — APPROVED** [review]
Awesome work here buddy! Thank you!

**Pull request merged**